package.json

```diff
@@ -30,7 +30,7 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx}": ["eslint --quiet --ext .js --ext .jsx ."]
+    "*.{js,jsx}": ["eslint --quiet --ext .js --ext .jsx --ignore-pattern '/extensions/*' ."]
```

Comment:
Same here. This doesnt make any sense. This rule should go into the eslintignore file.

Reply:
is correct